packages/app/src/cli/models/extensions/specifications/flow_template.ts

```diff
@@ -9,12 +9,22 @@ import fs from 'fs'
 
 const FlowTemplateExtensionSchema = BaseSchemaWithHandle.extend({
   type: zod.literal('flow_template'),
-  title: zod.string(),
-  description: zod.string(),
-  categories: zod.array(zod.string()),
-  require_app: zod.boolean(),
-  visible: zod.boolean(),
-  enabled: zod.boolean(),
+  name: zod.string(),
+  templates: zod
+    .array(
+      zod.object({
+        key: zod.string(),
+        name: zod.string(),
+        description: zod.string(),
+        categories: zod.array(zod.string()),
+        require_app: zod.boolean(),
+        listed: zod.boolean(),
+        enabled: zod.boolean(),
+      }),
+    )
+    // technically we'll be able to support more than one template per extension but restricting to one for now
+    .min(1)
+    .max(1),
```

Comment:
this makes more sense 👏 
No need to define `name` as it's already included in the BaseSchema